partition-pruning.md

```diff
@@ -0,0 +1,296 @@
+---
+title: Partition Pruning
+summary：Introduce application scenarios of TiDB partition pruning.
```

Comment:
Please do not use Chinese punctuation in an English doc, which might cause website build failure.